src/v2/Apps/Show/Components/ShowInfoLocation.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import { Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ShowInfoLocation_show } from "v2/__generated__/ShowInfoLocation_show.graphql"
+
+interface ShowInfoLocationProps {
+  show: ShowInfoLocation_show
+}
+
+export const ShowInfoLocation: React.FC<ShowInfoLocationProps> = ({ show }) => {
```

Comment:
Fully expect this component will require some finesse. I just have no idea what the logic should be for displaying what/linking to what, at the moment. Assuming we'll circle back to this at Q/A

Reply:
Agreed, at some point if we consolidate and figure out the standard we can move to MP, etc.